### PR TITLE
os: update links in coreos-hardening-guide

### PR DESCRIPTION
**Pull request by ElijahCaine**
Corrects an outdated link.

### CODE REVIEW

File: os/coreos-hardening-guide.md [hunk 1]
```diff
@@ -12,15 +12,15 @@ To disable sshd from listening you can stop the socket:
 systemctl mask sshd.socket --now
 ```
 
-If you wish to make further customizations see our [customize sshd guide](customizing-sshd.md).
+If you wish to make further customizations see our [customize sshd guide][etcd-ssh-guide].
```
Comment (crawford) [inline os/coreos-hardening-guide.md]: What is wrong with this link? (it also has nothing to do with etcd)

File: os/coreos-hardening-guide.md [hunk 2]
```diff
@@ -56,3 +56,6 @@ SocketGroup=root
 EOM
 systemctl daemon-reload
 ```
+
+[etcd-ssh-guide]: customizing-sshd.md
+[etcd-sec-guide]: https://github.com/coreos/etcd/blob/master/Documentation/op-guide/security.md
```
Comment (crawford) [inline os/coreos-hardening-guide.md]: Link against a tag or ref. Branches change over time and the links can break on their own.

File: os/coreos-hardening-guide.md (hunk 1 above)
Comment (joshix) [inline os/coreos-hardening-guide.md]: nothing; E. just made it into a reference link

**Comment by joshix** [comment]
@crawford Are we there yet?

**Comment by crawford** [comment]
Can you remove etcd from the reference name? Unless I'm missing something, that guide has nothing to do with etcd.

File: os/coreos-hardening-guide.md (hunk 1 above)
Comment (ElijahCaine) [inline os/coreos-hardening-guide.md]: Following the style guide. Bad git, good style.

**Comment by ElijahCaine** [comment]
@crawford Fixed. I wasn't actually reading the reference, just copyediting.

File: os/coreos-hardening-guide.md [hunk 3]
```diff
@@ -56,3 +56,6 @@ SocketGroup=root
 EOM
 systemctl daemon-reload
 ```
+
+[sshd-guide]: customizing-sshd.md
+[etcd-sec-guide]: https://github.com/coreos/etcd/blob/master/Documentation/op-guide/security.md
```
Comment (crawford) [inline os/coreos-hardening-guide.md]: Still cannot be a link to a branch.

**Pull request merged**